src/components/item/sharing/CCLicenseSelection.js

```diff
@@ -0,0 +1,178 @@
+    if (settings) {
+      setOptionValue(settings.ccLicenseAdaption);
+    }
+  }, [settings]);
+
+  if (isMemberLoading) return <Loader />;
+
+  const handleChange = (event) => {
+    setOptionValue(event.target.value);
+  };
+
+  const handleClick = () => {
+    const url = 'https://creativecommons.org/about/cclicenses/';
+    redirect(url, { OpenInNewTab: true });
+  };
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    updateCCLicense({
+      id: itemId,
+      name: itemName,
+      settings: { ccLicenseAdaption: optionValue },
+    });
+    handleClose();
+  };
+
+  return (
+    <>
+      <Typography variant="h6" className={classes.title}>
+        {t('Creative Commons License')}
+        <Tooltip
+          title={t('Need more info about CC License? Click here!')}
+          arrow
+        >
+          <IconButton aria-label="info" onClick={handleClick}>
+            <HelpIcon />
+          </IconButton>
+        </Tooltip>
+      </Typography>
+      <Typography variant="body1">
+        {t(
+          'All content published on Graasp Explorer does not allow commercial use.',
+        )}
+      </Typography>
+      <Typography variant="body1">
+        {t('Allow adaptations of your work to be shared?')}
+      </Typography>
+      <RadioGroup
+        aria-label="CC License"
+        name={t('CC License')}
+        value={optionValue}
+        onChange={handleChange}
+      >
+        <FormControlLabel
+          value={CC_LICENSE_ADAPTION_OPTIONS.ALLOW}
+          control={<Radio color="primary" />}
+          label={t('Yes')}
+        />
+        <FormControlLabel
+          value={CC_LICENSE_ADAPTION_OPTIONS.ALIKE}
+          control={<Radio color="primary" />}
+          label={t('Only if others share alike')}
+        />
+      </RadioGroup>
+      <Button
+        variant="outlined"
+        color="primary"
+        className={classes.button}
+        onClick={handleClickOpen}
+        disabled={disabled || !optionValue} // disable the button if no option is selected
+      >
+        {t('Submit')}
+      </Button>
+      <Dialog open={open} onClose={handleClose}>
+        <DialogTitle>{t('Confirm Your Submission')}</DialogTitle>
+        <DialogContent>
+          <DialogContentText>
+            {t('Attention: This Action is irrevertable!')}
+            <br />
+            {t(
+              'Once you submit your choice, you cannot remove selected Creative Commons License, or change to a more restricted choice.',
+            )}
+          </DialogContentText>
+        </DialogContent>
+        <DialogActions>
+          <Button onClick={handleClose} color="primary" autoFocus>
+            {t('Cancel')}
+          </Button>
+          <Button onClick={handleSubmit} color="primary">
+            {t('Confirm')}
+          </Button>
+        </DialogActions>
+      </Dialog>
```

Comment:
Maybe this can go in another component.
Also: irreversible exists in English right? It would be better than irrevertable^^